matter/wifi/rs911x/rsi_if.c

```diff
@@ -417,26 +416,27 @@ void wfx_rsi_task(void *arg)
      * Let's handle DHCP polling here
      */
     if (wfx_rsi.dev_state & WFX_RSI_ST_STA_CONNECTED) {
-      if ((now = xTaskGetTickCount()) > (last_dhcp_poll + pdMS_TO_TICKS(250))) {
+      if ((now = xTaskGetTickCount()) > (last_dhcp_poll + pdMS_TO_TICKS(DELAY_250MS))) {
 #if (CHIP_DEVICE_CONFIG_ENABLE_IPV4)
         uint8_t dhcp_state = dhcpclient_poll(sta_netif);
         if (dhcp_state == DHCP_ADDRESS_ASSIGNED && !hasNotifiedIPV4) {
           wfx_dhcp_got_ipv4((uint32_t)sta_netif->ip_addr.u_addr.ip4.addr);
           hasNotifiedIPV4 = true;
           if (!hasNotifiedWifiConnectivity) {
-            wfx_connected_notify(0, &wfx_rsi.ap_mac);
+            wfx_connected_notify(CONNECTION_STATUS_SUCCESS, &wfx_rsi.ap_mac);
```

Comment:
The naming here is good my issue is mostly with the value an how wfx_connected_notify and wfx_ip_changed_notify must be implemented.
CONNECTION_STATUS_SUCCESS =0  -> Success
IP_STATUS_FAIL = 0 -> failure

So in one case 0 indicate a success and the other 0 indicates a failure. It is not consistent.

Reply:
Changed CONNECTION_STATUS_SUCCESS = 1  ---> Success